additions/PythonExtensionsCollection/Folder/CFolder.py

```diff
@@ -0,0 +1,295 @@
+# **************************************************************************************************************
```

Comment:
Hi Holger,

this I would not expect here.
I would expect a general dependency on python-extension-collection which must be resolved before robotframework-extensions-collection can work.

Nevertheless I merge. Please consider this in future.

Thank you,
 Thomas